src/bin/upgrade_octree.rs

```diff
@@ -66,6 +66,23 @@ fn upgrade_version10(directory: &Path, mut meta: proto::Meta) {
     meta.write_to_writer(&mut buf_writer).unwrap();
 }
 
+fn upgrade_version11(directory: &Path, mut meta: proto::Meta) {
+    println!("Upgrading version 11 => 12.");
+    let mut octree = proto::OctreeMeta::new();
+
+    octree.set_bounding_box(meta.take_deprecated_bounding_box());
+
+    octree.set_resolution(meta.deprecated_resolution);
+    meta.deprecated_resolution = 0.0;
```

Comment:
What happens, if we don't set this to `0.0`, i.e. can we just leave it?

Reply:
I just copied the way it is in version 9 with deprecated level. btw take() also gets the contents by vale and replaces it with new(), so it sorts of clears the contents without having borrowing problems